ui/src/pages/Logs/Logs.tsx

```diff
@@ -175,9 +198,40 @@ const logsDisplayStyle = {
   overflow: 'hidden',
   fontFamily: 'monospace',
 };
+
+const colorArray: string[] = [
+  purple[100],
+  red[100],
+  blue[100],
+  teal[100],
+  green[100],
+  cyan[100],
+  orange[100],
+  yellow[100],
+  brown[100],
+  blueGrey[100],
+];
+// Keeps track of what color to assign to each unique container
+let colorCounter = 0;
+// Containers object holds key value pairs of {containerName: containerColor}
+const containerTracker: Record<string, string> = {};
```

Comment:
NIT: It is difficult to infer what the `containerTracker` stores based on the name.